integrationExamples/gpt/top-level-paapi/tl_paapi_example.html

```diff
@@ -0,0 +1,186 @@
+                        params: {
+                            unit: '538703464',
+                            response_template_name: 'test_banner_ad',
+                            test: true,
+                            delDomain: 'sademo-d.openx.net'
+                        }
+                    },
+
+                ],
+            }
+            ]
+        ;
+
+        var pbjs = pbjs || {};
+        pbjs.que = pbjs.que || [];
+
+        var googletag = googletag || {};
+        googletag.cmd = googletag.cmd || [];
+        googletag.cmd.push(function () {
+            googletag.pubads().disableInitialLoad();
+        });
+
+        pbjs.que.push(function () {
+            pbjs.setConfig({
+                debug: true,
+                paapi: {
+                    enabled: true,
+                    gpt: {
+                        autoconfig: false
+                    }
+                },
+                debugging: {
+                    enabled: true,
+                    intercept: [
+                        {
+                            when: {
+                                bidder: 'openx',
+                            },
+                            paapi() {
+                                return [
+                                    {
+                                        'seller': 'https://privacysandbox.openx.net',
+                                        'decisionLogicUrl': 'https://privacysandbox.openx.net/fledge/decision-logic-component.js',
+                                        'sellerSignals': {
+                                            'floor': 0.01,
+                                            'currency': 'USD',
+                                            'auctionTimestamp': new Date().getTime(),
+                                            'publisherId': '537143056',
+                                            'adUnitId': '538703464'
+                                        },
+                                        'interestGroupBuyers': [
+                                            'https://privacysandbox.openx.net'
+                                        ],
+                                        'perBuyerSignals': {
+                                            'https://privacysandbox.openx.net': {
+                                                'bid': 1.5
+                                            }
+                                        },
+                                        'resolveToConfig': true,
+                                        'sellerCurrency': 'USD'
+                                    }
+                                ];
+                            }
+                        }
+                    ]
+                },
+            });
+
+            pbjs.addAdUnits(adUnits);
+            pbjs.requestBids({
+                bidsBackHandler: sendAdserverRequest,
+                timeout: PREBID_TIMEOUT
+            });
+        });
+
+        function raa() {
```

Comment:
does it make sense to have some of this code in a submodule, which might end up being extremely compact on initial commit, but would provide the convenient logical abstraction that a user would pick the PAAPI submodule associated with their choice of TLS?

Reply:
I think so yes. A pub makes 2 decissions. 
1. I want to run PAAPI with prebid
2. I want to do it myself (pub sub module) or I want GPT to do it for me